backend/models/storage.go

```
@@ -1313,22 +1747,39 @@
 	}
+
 	if result.RowsAffected > 0 {
 		// record already exist, do nothing
+		slog.Info("updating existing repo cache",
+			"repoFullName", repoFullName,
+			"orgId", orgId,
+			"repoCacheId", repoCache.ID)
+
 		repoCache.DiggerConfig = configMarshalled
 		repoCache.DiggerYmlStr = diggerYmlStr
 		result = db.GormDB.Save(&repoCache)
```

golangci-lint (1.64.8)

SA4006: this value of `result` is never used

(staticcheck)